doc/overview/EDITING.md

```diff
@@ -1,125 +1,213 @@
+   2. the WM segmentation (`<subject_dir>/mri/wm.mgz`) and filled version (`<subject_dir>/mri/filled.mgz`) to initialize surfaces and
+   3. a brainmask (`<subject_dir>/mri/brain.finalsurfs.mgz`) to guide the positioning of the pial surfaces.
 
-   # Define data directory
-   datadir=/home/user/my_mri_data
-   fastsurferdir=/home/user/my_fastsurfer_analysis
+## Possible Edits
 
-   # Run FastSurfer
-   ./run_fastsurfer.sh --t1 $datadir/subjectX/t1-weighted-nii.gz \
-                    --sid subjectX --sd $fastsurferdir \
-                    --parallel --threads 4 --3T
+FastSurfer supports the following edits: 
+1. [Bias field corrected inputs](#bias-field-correction) (for improved image quality, not really an edit)
+2. [asegdkt_segfile](#asegdkt_segfile): `<subject_dir>/mri/aparc.DKTatlas+aseg.deep.mgz` via `<subject_dir>/mri/aparc.DKTatlas+aseg.deep.manedit.mgz`
+3. [Talairach registration](#talairach-registration): `<subject_dir>/mri/transforms/talairach.xfm` (overwrites automatic results from `<subject_dir>/mri/transforms/talairach.auto.xfm`)
+4. [White matter segmentation](#white-matter-segmentation): `<subject_dir>/mri/wm.mgz` and `<subject_dir>/mri/filled.mgz`
+5. [Pial placement](#pial-surface-placement): `<subject_dir>/mri/brain.finalsurfs.mgz` via `<subject_dir>/mri/brain.finalsurfs.manedit.mgz`
+
+Note, as FastSurfer's surface pipeline is derived from FreeSurfer, some edit options and corresponding naming schemes are inherited from FreeSurfer.
+
+## General process
+Most editing options require that you have processed the case and you are able to access and modify the files in that image's directory, i.e. `<subject_dir>`.
+Based on a quality inspection of results, segmentations and surfaces, one of the edits (see above) is chosen and performed.
+Then, the case is re-processed using the `--edits` flag, i.e. `run_fastsurfer.sh` is re-run (same FastSurfer command and FastSurfer version as before, but add `--edits`).
+Finally, you would check in another quality inspection, if the quality issues are resolved.
+
+For example (including the setup for native processing, see [Examples](EXAMPLES.md) for other processing options):
+
+```bash
+# Setup FASTSURFER and FREESURFER
+export FASTSURFER_HOME=/path/to/fastsurfer
+export FREESURFER_HOME=/path/to/freesurfer
+source $FREESURFER_HOME/SetUpFreeSurfer.sh
+
+# Define data directory
+export SUBJECTS_DIR=/home/user/my_fastsurfer_analysis
+
+# Run FastSurfer
+$FASTSURFER_HOME/run_fastsurfer.sh \
+    --sd $SUBJECTS_DIR --sid case_with_edits \
+    --t1 $SUBJECTS_DIR/case_with_edits/mri/orig/001.mgz \
+    --fs_license $FREESURFER_HOME/.license \
+    --edits # more flags as needed, e.g. --parallel --3T --threads 4
+```
+
+Note, a re-run of the segmentation pipeline, as in the command above, should not be harmful, but is only required if the [asegdkt_segfile](#asegdkt_segfile) was edited.
+There, we can usually skip the segmentation step with
```

Comment:
Maybe you mean: "Therefore, in most cases, we can skip ..."

Reply:
I think you actually wrote this sentence :).

```suggestion
Therefore, in most cases, we can skip the segmentation step with
```